src/internal/pachd/restore.go

```diff
@@ -28,7 +33,13 @@ func NewRestoreSnapshot(env RestoreSnapshotEnv, config pachconfig.PachdRestoreSn
 		setupStep{
 			Name: "HelloWorld",
 			Fn: func(ctx context.Context) error {
-				log.Info(ctx, "Hello, you would have just restored a snapshot")
+				tracker := track.NewPostgresTracker(env.DB)
+				storage := fileset.NewStorage(fileset.NewPostgresStore(env.DB), tracker, chunk.NewStorage(kv.NewMemStore(), nil, env.DB, tracker))
```

Comment:
This needs to be the real storage system, not the memstore.  Otherwise the restore can't download the database dump to apply.

(btw this is testpachd you're editing here, I think.)

Reply:
Updated to create a real storage now.

With respect to `testpachd`, this is called only from `src/server/cmd/pachd`.